src/components/__tests__/BannerRoutes.cy.js

```diff
@@ -5,10 +5,12 @@ import { rideToWorkByBikeConfig } from '../../boot/global_vars';
 import { routesConf } from '../../router/routes_conf';
 
 // colors
-const { getPaletteColor } = colors;
+const { getPaletteColor, changeAlpha } = colors;
 const grey10 = getPaletteColor('grey-10');
 const white = getPaletteColor('white');
 const primary = getPaletteColor('primary');
+const secondary = getPaletteColor('secondary');
+const secondaryOpacity = changeAlpha(secondary, 0.4);
```

Comment:
Secondary color opacity level value 0.4 is repeated across code. 

@maceksimon 

You can return back deleted global app variable `colorSecondaryOpacity`  as number type, and reuse it, please.

Without this it will be time consuming change secondary opacity level in many code files (because you define opacity value standalone in "every" module file).

Reply:
Yes, I see.
I brougth back the global config variable (`opacitySecondaryBackground` - to differentiate from color* vars with string color values).